engine/kv_engine.hpp

```diff
@@ -494,6 +498,11 @@ class KVEngine : public Engine {
   void cleanNoHashIndexedSkiplist(Skiplist* skiplist,
                                   std::vector<DLRecord*>& purge_dl_records);
 
+  double cleanSlotBlockOutDated(PendingPrugeFreeRecords& pending_clean_records,
```

Comment:
why not put these functions in reclaimer?

Reply:
Because these functions are almost called by engine. So I didn't move these to reclaimer. Reclaimer is responsible for dynamically adjust workers.

Reply:
named：cleaner